tests/rules/use-effects-lifecycle-interface.test.ts

```diff
@@ -122,6 +123,7 @@ ruleTester().run(path.parse(__filename).name, rule, {
       {
         output: stripIndent`
           import { OnInitEffects } from '@ngrx/effects';
+          import * as ngrx from '@ngrx/effects'
```

Comment:
The extra imports are here just to test that we correctly fix the import, right?
If that's that case and I'm being honest, then I do find it a bit confusing.
The reason is that I like to have the cases in here, as compact as possible.
But I also get that we want to test that it works 😅.

Reply:
Yes, that's it. I'm also part of the team that likes compact testing 😄

---

Taking advantage of the fact that we're talking about testing... I was looking at how `@typescript-eslint` does the tests and I found an interesting approach there: they don't mess with formatting literals, like these for example:

<img width="495" alt="Captura de Tela 2021-09-04 às 01 00 06" src="https://user-images.githubusercontent.com/11965907/132081706-a3e86f82-0e4b-48d3-a859-61474fa17cf2.png">

Source: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/tests/rules/no-unsafe-argument.test.ts

Do you think we could follow something similar here? I ask because I see that we have some formatting inconsistencies between the tests, which could be remedied this way. An example:

<img width="662" alt="Captura de Tela 2021-09-04 às 01 03 17" src="https://user-images.githubusercontent.com/11965907/132081769-9d371a49-0faa-4113-8217-8534b5abbaf7.png">

Reply:
Sure we can do that, I stole the `stripIndent` from the rjxs linter 😅
I'm don't have a big opinion, but I would say let's try to keep things consistent with the Angular one.
I'll review the other PRs on Monday.
Have a great weekend! ✨